### DISCUSSION TITLE
Aula de Fundamentos de Entity Framework

### DISCUSSION THREAD

**Discussion by LuizH1359** [balta-io/forum#352] [Tira Dúvidas] [answered]
Estava executando o primeiro comando de do crud que no caso criaria uma tag, porém notei que não estava funcionando e quando fui olhar dizia que estava tendo erro de certificação que o ssl não era emitido por uma entidade valida etc, então realizei o tutorial de erros e dei os comandos pra limpar e baixar o novo ssl e adicionei os parametros a connection string, porém quando coloco diz que sa não é um login válido, e se não coloco diz que o ssl foi emitido por uma entidade que não é válida. Poderia me Ajudar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @LuizH1359, como vai?
Tenta o seguinte:

```
dotnet dev-certs https --clean
dotnet dev-certs https --trust
```

Depois tenta executar o comando novamente por favor.

**Comment by LuizH1359** [comment]
Já tentei e não funcionou, se deixar a connection string como está aparece
Win32Exception: A cadeia de certificação foi emitida por uma autoridade que
não é de confiança.
e se colocar e aparece Microsoft.Data.SqlClient.SqlException: 'Login failed
for user 'sa'.' quando eu coloco na connection string
Trusted_Connection=False;
TrustServerCertificate=True;

Em qui., 29 de jun. de 2023 às 09:59, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Isso mesmo, ai precisa do TrusteServer como true...
Se você colocou isso e deu login inválido, é problema com o usuário e senha.

Se estiver usando Docker, segue nosso tutorial que fica igual do curso:
https://balta.io/blog/sql-server-docker

**Comment by LuizH1359** [comment]
Já até desinstalei e instalei novamente o docker porém continua dando erro
igual, fala que a certificação foi emitida por uma autoridade que não é de
confiança.

Em sex., 30 de jun. de 2023 às 09:52, Andre Baltieri <
***@***.***> escreveu:

**Reply by andrebaltieri** [reply]
Se estiver no Windows, tenta usar o SQL Server Express... estranho este erro de certificado mesmo refazendo os passos que comentei e adicionado o Trusted no Connection String.
